src/summary/LongitudinalTableVisualizer.jsx

```diff
@@ -51,27 +51,11 @@ export default class LongitudinalTableVisualizer extends Visualizer {
         return formattedData;
     }
     reorderRows = (clickedSection) => {
-        const data = [...this.state.data];
+        const data = this.state.data;
```

Comment:
I think I changed that because I realized that I wasn't changing data (or this.state.data) in any way so I thought it was okay to reference the state itself.  I can change it back though if it's good practice to clone even if you're not changing it.

Reply:
No, I think that makes sense, and I think it's good to stay as it is!